lms/envs/common.py

```diff
@@ -2758,6 +2759,7 @@
 MOBILE_APP_USER_AGENT_REGEXES = [
     r'edX/org.edx.mobile',
 ]
+MOBILE_APP_VERSION_REGEX = r'[0-9]+.[0-9]+.[0-9]+(.[0-9a-zA-Z]*)?'  # X.X.X.Y where X is a number and Y is alphanumeric
```

Comment:
To be clear, we're doing standard x.y.z for versions. However, since we're not presently a library with published interfaces, we're not following the ``semantic`` part of semver (where each part of the version represents what it for the API contract).